lib/vm.js

```diff
@@ -79,21 +79,21 @@ Script.prototype.runInNewContext = function(sandbox, options) {
   return this.runInContext(context, options);
 };
 
+function validateType(prop, propName, type = 'string') {
```

Comment:
The type is always a string and setting a default value is a minor overhead. So instead, please just directly use 'string' in this function.

Reply:
In that case, please change the function name to `validateString`.